### DISCUSSION TITLE
Importing other file types, e.g. CSS

### DISCUSSION THREAD

**Discussion by stuarthendren** [rafgraph/rollpkg#12] [Q&A] [answered]
Does Rollpkg support importing other file types? 

If not can it be made to do so (by default, with plugins or config) or would this be outside the intended scope of rollpkg and, if you need this functionality, you should just use Rollup directly with the plugins you need?

**Comment by rafgraph** [comment, marked as answer]
Hi, currently it is not possible and you should use Rollup directly with the plugins you need.

In the future I’d like to switch to using a default Rollup config that the user can add overrides too (e.g. add specific plugins) similar to the tsconfig, jest and eslint configs. Help/ideas for this are welcome.
